src/main/java/woofareyou/logic/Logic.java

```diff
@@ -26,7 +28,7 @@
     /**
      * Returns the AddressBook.
      *
-     * @see seedu.address.model.Model#getAddressBook()
+     * @see Model#getAddressBook()
```

Comment:
this might be:
```suggestion
     * @see woofareyou.model.Model#getAddressBook()
```
but i'm not entirely sure!

Reply:
yep!